src/status_im/contexts/wallet/common/wizard.cljs

```diff
@@ -0,0 +1,23 @@
+(ns status-im.contexts.wallet.common.wizard
+  (:require [status-im.contexts.wallet.send.flow-config :as wallet-flow]
+            [utils.re-frame :as rf]))
+
+(defn- wizard-find-next-screen
+  [db flow-config current-screen]
+  (->> flow-config
+       (filter (fn [{:keys [skip-step? screen-id]}]
+                 (and (not= screen-id current-screen)
+                      (not (and (fn? skip-step?) (skip-step? db))))))
+       first))
+
+(rf/reg-event-fx
+ :wallet/wizard-navigate-forward
+ (fn [{:keys [db]} [{:keys [current-screen flow-id start-flow?]}]]
+   (let [flow-config (case flow-id
+                       :wallet-flow wallet-flow/steps
+                       nil)
+         next-screen (wizard-find-next-screen db flow-config current-screen)]
```

Comment:
Maybe we should just pass flow-id to `wizard-find-next-screen` and have this let block there.

Reply:
Done